### ISSUE TITLE
Problems with testing

### ISSUE THREAD

**Issue by Crystalgirl211** [WuJie1010/Facial-Expression-Recognition.Pytorch#86]
Hello, on the CK + data set, I took the model with the highest accuracy (93.556%) of 10% cross-validation and then took a test in each of the CK + facial expression pictures, but none of the facial expressions were correctly identified , May I ask why this happens? Thank you

**Comment by Crystalgirl211** [comment]
I checked the code again，I am sure that  all preprocessing is aligned with your code ~ But the result is still the same as before.I am very confused.Can you give me some suggestions on this issue?Thank you ~

**Comment by WuJie1010** [comment]
if you test in all CK + data sets, it also includes the validation set. If the accuracy reaches 90%, it is impossible to have the problem that every picture is wrong. I think it must be a bug, please check your code again~

**Comment by Crystalgirl211** [comment]
您好，非常感谢您的回复，我已经仔细检查了代码，但是现在出现了一个新的问题：在fer2013数据集上，我发现您的代码中已经采取了各种防止过拟合的措施，但是我画了loss-epoch图，发现还是出现了过拟合现象（训练集上loss一直下降，accuracy一直上升，但是测试集上loss先下降后上升），请问怎么解决呢？再次感谢您的回复~祝好




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"WuJie"<notifications@github.com&gt;;
发送时间:&nbsp;2020年5月5日(星期二) 下午3:16
收件人:&nbsp;"WuJie1010/Facial-Expression-Recognition.Pytorch"<Facial-Expression-Recognition.Pytorch@noreply.github.com&gt;;
抄送:&nbsp;"玉纸尘"<2445155412@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [WuJie1010/Facial-Expression-Recognition.Pytorch] Problems with testing (#86)





 
if you test in all CK + data sets, it also includes the validation set. If the accuracy reaches 90%, it is impossible to have the problem that every picture is wrong. I think it must be a bug, please check your code again~

**Comment by WuJie1010** [comment]
> 您好，非常感谢您的回复，我已经仔细检查了代码，但是现在出现了一个新的问题：在fer2013数据集上，我发现您的代码中已经采取了各种防止过拟合的措施，但是我画了loss-epoch图，发现还是出现了过拟合现象（训练集上loss一直下降，accuracy一直上升，但是测试集上loss先下降后上升），请问怎么解决呢？再次感谢您的回复~祝好
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人:&nbsp;"WuJie"<notifications@github.com&gt;; 发送时间:&nbsp;2020年5月5日(星期二) 下午3:16 收件人:&nbsp;"WuJie1010/Facial-Expression-Recognition.Pytorch"<Facial-Expression-Recognition.Pytorch@noreply.github.com&gt;; 抄送:&nbsp;"玉纸尘"<2445155412@qq.com&gt;;"Author"<author@noreply.github.com&gt;; 主题:&nbsp;Re: [WuJie1010/Facial-Expression-Recognition.Pytorch] Problems with testing (#86) if you test in all CK + data sets, it also includes the validation set. If the accuracy reaches 90%, it is impossible to have the problem that every picture is wrong. I think it must be a bug, please check your code again~ — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

我虽然采用了过拟合策略，但不能完全避免过拟合~FER一般训练150+ epoch就可以达到最佳的结果~
